contracts/database/schema/schema.go

```diff
@@ -22,6 +22,8 @@ type Schema interface {
 	DropColumns(table string, columns []string) error
 	// DropIfExists Drop a table from the schema if exists.
 	DropIfExists(table string) error
+	// Extend the schema with given extend parameter.
+	Extend(extend *Extension) Schema
```

Comment:
How about `struct` instead of `pointer`? I think `pointer` is unnecessary.

```suggestion
	Extend(extend Extension) Schema
```